### ISSUE TITLE
sometimes it shows two windows

### ISSUE THREAD

**Issue by 532910** [vector-im/riotX-android#1642]
![image](https://user-images.githubusercontent.com/14617699/86860212-19afc600-c0cd-11ea-976c-0851fe5c88ed.png)

v0.91.3-beta

**Comment by lub** [comment]
I think this happens when you share something from another app

**Comment by ManDay** [comment]
Yes, can confirm. That way you can create infinitely many instances (?) of Riot. Each "Share with Riot" creates yet another one.

**Comment by 532910** [comment]
Moreover, sharing from F-Droid turns F-Droid into Element:
![image](https://user-images.githubusercontent.com/14617699/87954861-9a02fd80-cab5-11ea-8c5a-5b1619f46cb8.png)

**Comment by dipeve** [comment]
Not sure if related, but if you open a room shortcut from the home screen it creates a new instance of Element. Instead of replacing the current one if is one open. Version 1.0.10
![Uploading Screenshot_20201106-194042~2.png…]()

**Comment by 532910** [comment]
I believe this is another one #2360, like #2359

**Comment by riX2000** [comment]
This also the effect of keeping whatever was shared (if text) in the message field even after sending, probably because the new instance was started with the shared text as a parameter. The new instance has to be closed for this to go away, sending additional messages does not help.
